client/app/_types/house.ts

```diff
@@ -6,4 +6,5 @@ export type HouseType = {
   isExpensePerTime: boolean
   isAdmin: boolean
   rules: RuleType[]
+  houseMembers: { id: number; name: string }[]
```

Comment:
You also need to change GetMe response?

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/80/commits/4daa6acd5b6a3aa8c134517af63c50d976701887